managers/apt.go

```diff
@@ -12,7 +12,7 @@ func NewApt() *Manager {
 				"-y",
 			},
 			install: []string{
-				"install", "--no-install-recommends",
```

Comment:
Just out of curiosity, why?

Reply:
Because Ubuntu is meant to be used with recommends enabled. The policy on whether recommends are installed or not should be decided by the distribution (in their apt config), not by distrobuilder.